### PR TITLE
Spread default substations

### PR DESCRIPTION
**Pull request by Meklo**

### CODE REVIEW

File: src/main/java/org/gridsuite/geodata/server/DefaultSubstationGeoParameter.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@ToString
+public class DefaultSubstationGeoParameter {
+
+    static final double DEFAULT_SUBSTATION_RADIUS_OFFSET = 0.0001;
+    static final double DEFAULT_SUBSTATION_ANGLE_OFFSET = 25;
+
+    static final double TRIGGERING_THRESHOLD = 5;
```
Comment (SlimaneAmar) [inline src/main/java/org/gridsuite/geodata/server/DefaultSubstationGeoParameter.java:26]: decluttering_treshold ?

File: src/main/java/org/gridsuite/geodata/server/DefaultSubstationGeoDataByCountry.java [hunk 2]
```diff
@@ -18,6 +18,7 @@
 import java.io.UncheckedIOException;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.Set;
 
 /**
  * @author Nicolas Noir <nicolas.oir at rte-france.com>
```
Comment (SlimaneAmar) [inline src/main/java/org/gridsuite/geodata/server/DefaultSubstationGeoDataByCountry.java:24]: Typo ?

**Review by SlimaneAmar — APPROVED** [review]

File: src/main/java/org/gridsuite/geodata/server/DefaultSubstationGeoParameter.java (hunk 1 above)
Comment (Meklo) [reply src/main/java/org/gridsuite/geodata/server/DefaultSubstationGeoParameter.java:26]: Agreed, sounds more precise

File: src/main/java/org/gridsuite/geodata/server/DefaultSubstationGeoDataByCountry.java (hunk 2 above)
Comment (Meklo) [reply src/main/java/org/gridsuite/geodata/server/DefaultSubstationGeoDataByCountry.java]: Fixed it

**Review by bendaamerahm — APPROVED** [review]

**Pull request merged**